README.md

```diff
@@ -22,7 +22,7 @@ There are different levels to this assignment, level 1 being the easiest, level
 
 * Fetch the tax rates by year: http://localhost:5000/tax-calculator/brackets/<2019|2020|...>
 
-_Warning: this endpoint has gotchas_
+_Warning: this endpoint has gotchas, which require special handling in your code._
```

Comment:
How about: 
`Warning: This endpoint sometimes returns bad responses, handle as appropriate.`